### ISSUE TITLE
Metadata API

### ISSUE THREAD

**Issue by marks** [attachmentsme/ruby-box#96]
Any plans to support the Metadata API? See http://developers.box.com/metadata-api/

Thanks,

Mark

**Comment by rsciandra** [comment]
Was wondering about this myself?!

**Comment by rposborne** [comment]
:+1:

**Comment by tblos** [comment]
Just an FYI, there is basic metadata support in our new ruby sdk: https://github.com/cburnette/boxr

**Comment by rposborne** [comment]
@tblos is that repo Officially sponsored?

**Comment by tblos** [comment]
Yes

**Comment by mike-yesware** [comment]
@tblos We no longer use this gem, which has lead to it no longer being supported. If you have no objection, I will suggest this in our README.

Any chance it will be moving from your account to the Box org?

**Comment by cburnette** [comment]
Yes, this repo will be transferred to the Box github account sometime in the next few weeks.  I would go ahead and link to it and then I can ping you when we update the repo location.

thanks,
Chad
